datarepo-clienttests/src/main/java/runner/config/ApplicationSpecification.java

```diff
@@ -0,0 +1,43 @@
+package runner.config;
+
+public class ApplicationSpecification implements SpecificationInterface {
+  public int maxStairwayThreads = 20;
+  public int maxBulkFileLoad = 1000000;
+  public int loadConcurrentFiles = 4;
+  public int loadConcurrentIngests = 2;
+  public boolean inKubernetes = false;
+  public long loadHistoryCopyChunkSize = 1000;
+  public long loadHistoryWaitSeconds = 2;
+
+  public ApplicationSpecification() {}
+
+  /** Validate the application specification read in from the JSON file. */
+  public void validate() {
+    if (maxStairwayThreads <= 0) {
```

Comment:
Might we want some generous maximums to prevent resource exhaustion or catch mistakes? For example, having `maxStairwayThreads < 1000`.

Reply:
I think any sanity checking should be in the application, not the test runner. Anyway, I think we want to do some testing first to figure out what a good range is.